heron/spi/src/java/com/twitter/heron/spi/utils/SchedulerUtils.java

```diff
@@ -431,16 +434,45 @@ public static boolean curlAndExtractPackage(
   /**
    * Get a resource that requires the maximum amount of ram, cpu and disk
    */
-  public static Resource getMaxRequiredResource(PackingPlan packingPlan) {
+  public static PackingPlan getHomogenizedContainerPlan(PackingPlan packingPlan) {
```

Comment:
I have already created a unit test for this method, see SchedulerUtilsTest.updatesPackingPlanWithUniformSizedContainers#L180